storage/src/factory.rs

```diff
@@ -193,6 +205,13 @@ impl BlobFactory {
             ))),
         }
     }
+
+    fn check_cache_stat(&self) {
+        let mgrs = self.mgrs.lock().unwrap();
```

Comment:
Blocking on `sync::Mutex` in async context is not a good choice. Can we use tokio mutex?
Besides, for `unwrap`, we usually comment it why it is safe to call unwrap()